themes/Afterglow.conf

```diff
@@ -1,7 +1,7 @@
 background            #202020
 foreground            #d0d0d0
 cursor                #d0d0d0
-selection_background  #303030
+selection_background  #eecb8b
```

Comment:
From the original theme design, I can see that the color for selection background is `#5A647E` ([see here](https://github.com/YabataDesign/afterglow-theme/blob/master/Afterglow.tmTheme#L23))